webapp/src/ts/services/feedback.service.ts

```diff
@@ -30,9 +31,20 @@ export class FeedbackService {
   };
 
   private logIdx = 0;
+  private lastErrorMessage;
   private readonly LEVELS = ['error', 'warn', 'log', 'info'];
   private readonly LOG_LENGTH = 20;
   private readonly logCircularBuffer = new Array(this.LOG_LENGTH);
+  // List of Error messages to not automatically log to feedback
+  // Can be a lower-cased partial string or a regular expression
+  private readonly NO_FEEDBACK_MESSAGES = [
```

Comment:
i'm worried about bugs like https://github.com/medic/config-muso/issues/9 where users generated 100k+ feedback docs... mitigating such issues without a core update would be great, it could be really nice if this list was augmentable/configurable via app_settings (or alike)

Reply:
An alternative is to keep a local history of more than just last message and not allow more than X duplicates to be submitted in one session.

Reply:
This way, there's no way to accidentally disable important feedback docs from being created at all.
We can add a telemetry entry for when the X limit is exceeded, so there is an idea that there's a feedback doc landslide.

Reply:
would you ever consider moving away from the feedback doc format (one error per doc) and move to a daily error summary doc? like a list of "unique errors" + timestamps/logs for first x instances?

Reply:
how hard would that be to query?

Reply:
this will add a delay between the error and when it makes it to the server, like telemetry documents. it can also be affected by bugs preventing the aggregation - like telemetry documents. it's a risk to add this complexity, it depends how much we value this information or not, I suppose.

Reply:
i believe it would be pretty easy to write equivalent postgres queries. i can contribute that if we want a path like that.

i agree with your summary of risks. i suspect users value this information highly. i love the idea that people responding to this data within 24hrs; i don't think that is our reality today.  that said, feels bad to make a technology choice which blocks same-day response from becoming possible.

maybe we stay with the hardcoded list for now? perhaps we can prevent an initial flood of feedback docs via testing?

Reply:
I've enjoyed this exchange!
 
To be honest, I am expecting a flood of feedback docs, both legitimate (actual flaws in either software or config) and illegitimate (noise). I think it's a big effort to try to anticipate potential sources, and not something that can be realistically done exhaustively. 
I'm inclined to push this change as-is and monitor projects that launch/upgrade to this version closely to see what they produce. 

Open to options and other ideas!